.travis-ci.sh

```diff
@@ -15,6 +15,11 @@ elif [[ $TASK = 'htmlproofer-external' ]]; then
   bundle exec jekyll build
   bundle exec htmlproofer $HTMLPROOFER_OPTIONS --external_only
 elif [[ $TASK = 'eip-validator' ]]; then
+  ls | grep 'eip-' | grep -v 'eip-template.md' || EXIT=$?
```

Comment:
This looks like it works, but generally piping the output of `ls` isn't a great idea. If you're trying to count the number of files with some name, it makes more sense to just do that directly semantically:

```sh
if [[ $(find . -name 'eip-*' | wc -l) -ne 1 ]]; then
  echo "only 'eip-template.md' should be in the root"
  exit 1
fi
```

(Also prefer ``[[ ]]` over `[ ]` [in bash](https://stackoverflow.com/a/669486).)

Reply:
This looks great, thanks @adlerjohn. Will update tomorrow.